assets/common/rclone-logs.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+. /.jelenv
+
+nodeId=$(hostname | sed 's/^[[:alpha:]]\+\([[:digit:]]\+\).*/\1/' | tr [A-Z] [a-z])
```

Comment:
why convert uppercase to lowercase (from an hostname) to get the node ID ?

Also I gess something like this will be more readable
```suggestion
nodeId=$(hostname | sed -r 's/^node([0-9]+)-.*$/\1/'
```

Reply:
Fixed